tools/run_onnx.cc

```diff
@@ -558,10 +558,7 @@ void RunMain(const std::vector<std::string>& argv) {
     std::string test_path = args.get<std::string>("test");
 
     g_quiet = args.exist("quiet");
-    if (!onnx_path.empty() && !test_path.empty()) {
-        std::cerr << args.usage() << std::endl;
-        QFAIL() << "Specifying both --onnx and --test is invalid!";
-    } else if (onnx_path.empty() && test_path.empty()) {
+    if (onnx_path.empty() && test_path.empty()) {
```

Comment:
What is the intention of this change?

Reply:
I want to use test data with the quantized model separately from the original non-quantized model

Reply:
Ah, that sounds very useful. I'm not sure why I haven't thought of this, thanks!